thresher/models.py

```diff
@@ -101,13 +107,27 @@ def __unicode__(self):
         return "Topic %s in Analysis Type %s" % (self.name, self.analysis_type.name) 
 
 # The question in a given topic
-class Question(models.Model):
+class QuestionUnderTopic(models.Model):
     # an id within the given topic
-    question_id = models.IntegerField()
+    question_id = models.OneToOneField(QuestionContent)
 
     # The topic this question belongs to
-    topic = models.ForeignKey(Topic, related_name="questions")
+    topic_id = models.ForeignKey(Topic, related_name="related_questions")
+
+    # The order of the question compared to other questions under the same topic
+    order = models.IntegerField()
     
+    class Meta:
+        unique_together = ("question_id", "topic_id")
+
+    def __unicode__(self):
+        return "Question %d in Topic %s" % (self.question_id, self.topic_id.name)
+
+# Question itself
+class QuestionContent(models.Model):
+    # The question id the content is related to
+    question_id = models.IntegerField()
```

Comment:
What's the rationale behind splitting the Question into 2 separate models? And what is question_id referring to in this context (is it different than order?)